src/main/kotlin/ua/anastasiia/finesapp/repository/MongoFineRepository.kt

```diff
@@ -0,0 +1,128 @@
+package ua.anastasiia.finesapp.repository
+
+import org.bson.types.ObjectId
+import org.springframework.data.mongodb.core.MongoTemplate
+import org.springframework.data.mongodb.core.aggregate
+import org.springframework.data.mongodb.core.aggregation.Aggregation
+import org.springframework.data.mongodb.core.aggregation.Aggregation.group
+import org.springframework.data.mongodb.core.aggregation.Aggregation.match
+import org.springframework.data.mongodb.core.aggregation.Aggregation.unwind
+import org.springframework.data.mongodb.core.find
+import org.springframework.data.mongodb.core.findAll
+import org.springframework.data.mongodb.core.findOne
+import org.springframework.data.mongodb.core.query.Criteria
+import org.springframework.data.mongodb.core.query.Query
+import org.springframework.data.mongodb.core.query.Update
+import org.springframework.stereotype.Repository
+import ua.anastasiia.finesapp.dto.TotalFineSumResponse
+import ua.anastasiia.finesapp.entity.MongoFine
+import ua.anastasiia.finesapp.entity.MongoFine.Companion.COLLECTION_NAME
+import ua.anastasiia.finesapp.util.findAndModify
+import ua.anastasiia.finesapp.util.findAndRemove
+import java.time.LocalDate
+
+@Repository
+@Suppress("TooManyFunctions")
+class MongoFineRepository(val mongoTemplate: MongoTemplate) {
+    fun getAllFines(): List<MongoFine> = mongoTemplate.findAll<MongoFine>()
+    fun getAllFinesInLocation(longitude: Double, latitude: Double): List<MongoFine> {
```

Comment:
I think it's hard to have fines at an exact location. you can search fines in some areas (like within 1km radius of location)